WalletWasabi/Tor/TorMonitor.cs

```diff
@@ -144,26 +146,45 @@ await foreach (TorControlReply reply in client.ReadEventsAsync(cancellationToken
 	/// <inheritdoc/>
 	protected override async Task ActionAsync(CancellationToken token)
 	{
-		if (TorHttpPool.TorDoesntWorkSince is { }) // If Tor misbehaves.
+		if (TorHttpPool.TorDoesntWorkSince is not null) // If Tor misbehaves.
 		{
 			TimeSpan torMisbehavedFor = DateTimeOffset.UtcNow - TorHttpPool.TorDoesntWorkSince ?? TimeSpan.Zero;
 
 			if (torMisbehavedFor > CheckIfRunningAfterTorMisbehavedFor)
```

Comment:
Can we remove all the conditions after this, that addresses if we should try clearnet? 
If it is not working for a long time just try clearnet no matter what.

Reply:
(I do not take Tor process restart mechanism into account, that can mess my idea up - maybe not)